app/components/sort_button_component.rb

```diff
@@ -1,5 +1,5 @@
 class SortButtonComponent < ApplicationComponent
-  attr_reader :title, :name, :value
+  attr_reader :title, :name, :value, :url
```

Comment:
Is this change needed? I don't see `url` being used anywhere.

Reply:
Fixed this in a recent commit, it was a residual for one of my tests. Just missed in list of changes.